src/main/resources/ApplicationResources_es.properties

```diff
@@ -1520,3 +1520,44 @@ manage.autopublish.june_december=June/December
 
 # Struts2 overrides
 struts.messages.upload.error.SizeLimitExceededException=¡El archivo es demasiado grande\! El tamaño máximo permitido es de {0} bytes, su archivo es de {1} bytes. Para solucionar este problema, intente comprimir el archivo (con zip o gzip), cargar los datos en una de las muchas bases de datos soportados por el IPT o dividiendo el archivo. Por favor consulte el manual del usuario para obtener más información.
+
+####
+# FAQ Page
+####
+faq.title=FAQ - Preguntas Frecuentes
+faq.intro=En esta secci\u00F3n usted podr\u00E1 encontrar respuestas a las preguntas m\u00E1s frecuentes acerca del Catalogador de Informaci\u00F3n Biol\u00F3gica del Instituto Humboldt (CEIBA). Por favor revise el contenido de esta p\u00E1gina antes de contactar a la Infraestructura Institucional de Datos (I2D) del Instituto Humboldt.
```

Comment:
I suggest changing these texts to content in the file src/main/webapp/WEB-INF/pages/faq.ftl in [the commit No.13 in IPT-Ceiba](https://github.com/I2DHumboldt/IPT-Ceiba/commit/7d7b20e12b781283a17bbceb4289ba71deb384c7)